frontend/src/components/left-column.tsx

```diff
@@ -94,8 +94,16 @@ class LeftColumn extends React.Component<LeftColumnProps, LeftColumnState> {
     })
   }
 
+  private getDataContentType(ce: CloudEvent): string {
+    if (ce.datacontenttype == null) {
+      return "undefined datacontenttype"
```

Comment:
```suggestion
      return 'undefined'
```
or
```suggestion
      return 'null'
```

Reply:
This is text which will be written in the GUI. If user will be given "undefined" or "null", he won't have idea what is undefined or null.

Reply:
Ah, yes. However, I think typing `undefined datacontenttype` is ugly. I rather don't show anything there - skip the whole field.

Reply:
I don't think it matters if it is ugly. I think what matters is the information value given to the user. If There will be empty string, he won't know why. Using "undefined" also isn't very informative - you need to know this particular section is for datacontenttype. If there would be header in the section, then it would be OK to give "undefined".

Reply:
Fixed.